backend/requirements.txt

```diff
@@ -14,6 +15,7 @@ flake8==5.0.4
 Flask==1.1.2
 Flask-Cors==3.0.10
 Flask-Migrate==3.0.1
+Flask-SQLAlchemy==2.5.1
```

Comment:
Is it necessary to add these new dependencies? I'm not sure if they should be required since I don't see any new imports, but let me know if it breaks otherwise?